apps/wallet-dashboard/app/(protected)/home/page.tsx

```diff
@@ -25,15 +25,15 @@ function HomeDashboardPage(): JSX.Element {
         <main className="flex flex-1 flex-col items-center space-y-8 py-md">
             {connectionStatus === 'connected' && account && (
                 <>
-                    <div className="home-page-grid-container h-full w-full">
+                    <div className="home-page-grid-container w-full">
```

Comment:
Its weird for an account without tokens. All the sections should be together, right?
![image](https://github.com/user-attachments/assets/132acf59-434b-48b3-b51a-ea2d8e499f8b)